### ISSUE TITLE
Upgrade from C99 -> C100 on `macOS x64` via`1.38.x - Nightly` (Checking C100)

### ISSUE THREAD

**Issue by kjozwiak** [brave/brave-browser#21739]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromoium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [x] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [x] `CertificateRevocation`
  - [x] `cffkpbalmllkdoenhmdmpbkajipdjfam`: `rs-ABPFilterParserData.dat` & `regional_catalog.json` (AdBlock)
  - [x] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
  - [x] `jicbkmdloagakknpihibphagfckhjdih`: `speedreader-updater.dat`
  - [x] `oofiananboodjbbmdelgdommihjbkfag`: HTTPSE
  - [x] `Safe Browsing`
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers. We basically want to make sure that components are working as they regressed in C98 when we first merged into `master`.

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Rewards
    - [x] Wallet balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Wallet panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by stephendonner** [comment]
Currently blocked until https://github.com/brave/brave-browser/issues/21743 is resolved.

**Comment by stephendonner** [comment]
Verified `PASSED` using 

Brave | 1.38.52 Chromium: 100.0.4896.46 (Official Build) nightly (x86_64)
-- | --
Revision | 5ca33821b2211805855c77d334353d27c616a7ca-refs/branch-heads/4896@{#584}
OS | macOS Version 11.6.3 (Build 20G415)

I believe https://github.com/brave/brave-browser/issues/21743 ended up being due to an incorrect signing key being used in Jenkins; will work with @emerick to get the issue updated.
